core/templates/components/summary-tile/topic-summary-tile.directive.html

```diff
@@ -10,16 +10,18 @@
     </div>
     <div class="topic-details" ng-style="{'background-color': $ctrl.getDarkerThumbnailBgColor()}" ng-class="{'mask-effect': !$ctrl.isPublished()}">
       <span class="topic-name"> <[$ctrl.getTopicSummary().getName()]> </span>
-      <span class="topic-story-count">
+      <span class="topic-story-count" ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() > 0">
         <[$ctrl.getTopicSummary().getCanonicalStoryCount()]>
         <span ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() === 1" translate="I18N_TOPIC_VIEWER_STORY"></span>
-        <span ng-if="$ctrl.getTopicSummary().getCanonicalStoryCount() !== 1" translate="I18N_TOPIC_VIEWER_STORIES"></span>
       </span>
     </div>
   </a>
 </div>
 <style>
   .faded-topic-card {
+    background: rgb(93, 109, 108)!important;
+    /*important is to overeide the default background color
+      for the unpublished card*/
```

Comment:
Hm this seems a bit icky. We use the brightness filter to achieve the "coming soon" effect. Seems like after you center justify the image, the background is not using the filter. So let's do this instead:

https://github.com/oppia/oppia/blob/develop/core/templates/components/summary-tile/topic-summary-tile.directive.html

See how we have used the "filter" rule in a bunch of places here? Let's add an additional rule called `backdrop-filter` in all these places that takes the same values as `filter`. You can read more about `backdrop-filter` rule [here](https://css-tricks.com/almanac/properties/b/backdrop-filter/).

Reply:
Done.